### PR TITLE
ZTE-SH-CN-static-pod

### PR DESCRIPTION
**Pull request by houjun41544**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5449)

### CODE REVIEW

**Comment by houjun41544** [comment]
@zhangxiaoyu-zidif

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+approvers:
+- jsafrane
+title: 静态Pods
+---
+
+**如果你正在运行Kubernetes集群并且使用静态pods在每个node上起一个pod，那么最好使用[DaemonSet](/docs/concepts/workloads/controllers/daemonset/)!**
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: node可以翻译

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+*静态pods*直接由特定node上的kubelet进程来管理，不通过master上的APIserver。静态pod不关联任何replicationcontroller，它由kubelet进程自己来监控，当pod崩溃时重启该pod。对于静态pod没有健康检查。静态pod始终绑定在某一个kubelet，并且始终运行在同一个node上。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: master可以译为 主控节点

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Kubelet自动为每一个静态pod在Kubernetes API server上创建一个所谓的mirror pod，因此可以在API server查询到该pod，但是不被API server控制（例如不能删除）。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: API server -> API 服务器

Comment (tengqm) [reply cn/docs/tasks/administer-cluster/static-pod.md]: 所谓的mirror pod -> 镜像Pod（Mirror Pod）

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+## 静态pod创建
+
+静态pod有两种创建方式：用配置文件或者通过HTTP。
+
+### 配置文件
+
+配置文件就是放在特定目录下的标准的json或yaml格式的pod定义文件。用`kubelet --pod-manifest-path=<the directory>`来启动kubelet进程，kubelet将会周期扫描<the directory>这个目录，根据这个目录下appear/disappear的yaml/json文件来create/delete静态pod。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: json -> JSON, yaml -> YAML
appear/disappear -> 出现或消失
create/delete -> 创建／删除

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+下面例子用静态pod的方式启动一个nginx的webserver：
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: webserver -> Web 服务器

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    ```
+
+3. 配置node上的kubelet使用这个目录，kubelet启动时增加`--pod-manifest-path=/etc/kubelet.d/`参数。
+	如果是Fedora系统，在Kubelet配置文件/etc/kubernetes/kubelet中添加下面这行：
+	
+    ```
+    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --pod-manifest-path=/etc/kubelet.d/"
+    ```
+
+    Instructions for other distributions or Kubernetes installations may vary.
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: 漏翻

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+4. 重启kubelet。如果是Fedora系统，就是：
+
+    ```
+    [root@my-node1 ~] $ systemctl restart kubelet
+    ```
+
+## 通过HTTP创建静态Pods
+
+Kubelet周期从--manifest-url=<URL>参数指定的地址下载一个文件，并且把它翻译成json/yaml格式的pod定义。之后操作方式与--pod-manifest-path=<directory>相同，kubelet会不时的重新下载该文件，当文件变化时相应的终止或启动静态pod（如下）。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: 周期 -> 周期性地
一个文件 -> 文件
json -> JSON
yaml -> YAML
之后 -> 此后的
不时的 -> 不时地
相应的 -> 对应地

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+如果我们查看Kubernetes API server（运行在主机 `my-master`），可以看到这里创建了一个新的mirror-pod：
+
+```shell
+[joe@host ~] $ ssh my-master
+[joe@my-master ~] $ kubectl get pods
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          2m
+```
+
+静态pod的Labels会传递给mirror-pod，可以用来过滤或筛选。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: labels要翻译
mirror pod -> 镜像Pod

File: cn/docs/tasks/administer-cluster/static-pod.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+```shell
+[joe@my-master ~] $ kubectl delete pod static-web-my-node1
+pods/static-web-my-node1
+[joe@my-master ~] $ kubectl get pods
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          12s
+```
+
+返回`my-node1`主机，我们尝试手动终止container，可以看到kubelet很快就会自动重启container。
```
Comment (tengqm) [inline cn/docs/tasks/administer-cluster/static-pod.md]: container 需要翻译

**Comment by houjun41544** [comment]
Modified.Please review again.Thinks. @tengqm

**Pull request merged**